docs/source/release/v6.2.0/index.rst

```diff
@@ -4,9 +4,9 @@
 Mantid 6.2.0 Release Notes
 ===========================
 
-.. figure:: ../../images/ImageNotFound.png
+.. figure:: ../../images/ReleaseNotesIndex_v6_2.png
```

Comment:
This is a nice image. Were we not able to include the workspace calculator in the end?

Reply:
Yes, I think it is worth to include, perhaps at full stretch in the bottom.

Reply:
I've added the workspace calculator image in but I haven't included it at full stretch as it distorts horribly.

Reply:
Ahh, I meant to stretch the widget in workbench before taking the screenshot, but taking the existing screenshot is fine as well.